app/controllers/graphql_controller.rb

```diff
@@ -0,0 +1,11 @@
+class GraphqlController < ApplicationController
+  def execute
```

Comment:
are we going to have any authentication/authorization here?

So far in other services i've been using Grape layer authentication to decode JWT and find `user_id` and `role` and pass it as `context` to resolver [exmaple](https://github.com/artsy/gravity/blob/master/app/api/graph/root_endpoint.rb#L21)

Graphql ruby also allows you to define `middleware` for each schema [example](https://github.com/artsy/lewitt-api/blob/master/app/api/graph/schema.rb#L5)

Reply:
So I didn't implement it, but https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/13601/diffs looks interesting.